app/components/users/hover_card_component.sass

```diff
@@ -0,0 +1,23 @@
+// When displaying the hover card on top of an existing modal, we must
+// set the z-index so that the hover card stays in the front.
+.op-user-share-modal-overlay
+  position: absolute
+  z-index: 9600
+
+// Correct the z-index of the regular hover card container so that it is above user auto completers
+.spot-modal-overlay:has(.op-user-hover-card)
+  z-index: 9600
```

Comment:
Despite the comments I don't really get why this is needed (or rather why other implementations with a custom overlay don't need it). Maybe we can shortly talk about this.